schemas/maas-backend/customers/personalData.json

```diff
@@ -56,6 +56,11 @@
     "subscriberType": {
       "description": "Subscriber Type",
       "type": "string"
+    },
+    "profileImageUrl": {
+      "type": "string",
+      "minLength": 10,
+      "maxLength": 1024
     }
```

Comment:
Whether we do it in this PR or not doesn't matter so much. But in general it would be nice if the objects always contained some named types rather than primitive types.

Reply:
> But in general it would be nice if the objects always contained some named types rather than primitive types.

I agree, but let's do that in another PR